src/whisp/whisp.service.ts

```diff
@@ -72,9 +72,9 @@ export class WhispService {
       // ... wait until it is loaded ...
       filePromise.file.then((file) => {
         if (
-          file
-          && file.createReadStream
-          && {}.toString.call(file.createReadStream) === '[object Function]'
+          file &&
+          file.createReadStream &&
+          {}.toString.call(file.createReadStream) === '[object Function]'
```

Comment:
```suggestion
          {}.toString.call(file?.createReadStream) === '[object Function]'
```
Je pense que ça marche et tu gagnes en complexité

Reply:
Hum je sais pas trop comment tester cette partie là de l'appli encore, du coup je vais peut-être laisser comme ça pour l'instant (ce sont des changements faits par le linter) et voir plus tard quand je travaillerai dessus !